### PR TITLE
adding new exciting fields to the project page

### PR DESCRIPTION
**Pull request by Galithil**
this adds three fields to the sample view in the project page, based on the corresponding entry in the project db

### CODE REVIEW

File: run_dir/design/project_samples.html [hunk 1]
```diff
@@ -330,7 +330,7 @@ <h4 class="panel-title"><a data-toggle="collapse" data-parent="#undefined_fields
           <p>Choose which columns you would like to see in the project's Samples table.
             Click a heading to select / deselect all checkboxes in that group.</p>
           <div class="row">
-            <div class="col-sm-4" id="basic-columns">
+            <div class="col-sm-3" id="basic-columns">
```
Comment (ewels) [inline run_dir/design/project_samples.html]: Do you need to make these narrower if there isn't a new column in this top row?

**Comment by ewels** [comment]
Nice!

File: run_dir/design/project_samples.html (hunk 1 above)
Comment (Galithil) [inline run_dir/design/project_samples.html]: not really, no.

**Comment by Galithil** [comment]
Deployed on stage

**Comment by ewels** [comment]
Cool, looks good! Merging.

**Comment by remiolsen** [comment]
Found a problem for one project with multiple values for each sample row, ie. was run on several lanes (proj 1171)

**Pull request merged**